.cicd/release.sh

```diff
@@ -15,4 +15,14 @@ set -eou pipefail
   helm package .
 
   mv "weaviate-$VERSION.tgz" "weaviate.tgz"
+
+  echo "Add chart version $VERSION to GitHub Pages"
+  cd ..
+  mkdir -p files-to-gh-pages
+  echo $(pwd)
+  helm package weaviate -d files-to-gh-pages
+  cp README.md files-to-gh-pages
+  cd files-to-gh-pages
+  helm repo index .
+  ls -ltr
```

Comment:
is this command needed?

Reply:
I think this is to make sure everything is installed where it is supposed to. I do not think it is used besides debugging.

Reply:
ok, let's leave it